lib/text_count_util/counter.ex

```diff
@@ -0,0 +1,36 @@
+defmodule TextCountUtil.Counter do
+  use GenServer
+
+  alias TextCountUtil.{CounterDynamicSupervisor, CounterRegistry}
+
+  def start_link(line_number) do
+    name = {:via, Registry, {CounterRegistry, line_number}}
```

Comment:
I like the idea of naming processes with the line numbers, but do we actually need a named process here?

Reply:
`name = {:via, Registry, {CounterRegistry, ''}}` is right?

Reply:
Why not? Any term works, empty list is a great term.

But why would you need to name them at all?

Reply:
> But why would you need to name them at all?
You mean quite unused Registry for name processes in this case?
I deleted the Registry, I used to think it wouldn’t work without it.